druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/JDBCExtractionNamespaceCacheFactoryWithLeaderAndFollower.java

```diff
@@ -329,6 +329,10 @@ public Timestamp updateLocalCache(final JDBCExtractionNamespace extractionNamesp
      * @return
      */
     synchronized Producer<String, byte[]> ensureKafkaProducer(Properties kafkaProperties) {
+        kafkaProperties.put(ProducerConfig.ACKS_CONFIG, kafkaProperties.getProperty("retries", "0"));
+        kafkaProperties.put(ProducerConfig.RETRIES_CONFIG, kafkaProperties.getProperty("acks", "0"));
```

Comment:
The above two look swapped. You are getting retries for ACKS config.

Reply:
Thanks!